### ISSUE TITLE
Issue since latest version in docker when password enabled

### ISSUE THREAD

**Issue by gege-fr** [coderaiser/cloudcmd#310]
Hi,

When using the latest docker container , cloud commander crashes with following error if auth is enabled (i'm using ENV variables AUTH/USER/PASSWORD) 

```
criton is not a function Promise {
  <rejected> TypeError: criton is not a function
      at getPassword (file:///usr/src/app/bin/cloudcmd.mjs:235:12)
      at async main (file:///usr/src/app/bin/cloudcmd.mjs:204:28)
}
```

**Comment by coderaiser** [comment]
Thanks a lot :), fixed with 8957085, landed in [v15.0.2](https://github.com/coderaiser/cloudcmd/releases/tag/v15.0.2) 🎉 .
Is it works for you?

**Comment by gege-fr** [comment]
Yes, it works.

Thanks a lot !
